client/src/pages/ProfileDetails/ProfileDetails.tsx

```diff
@@ -23,25 +59,24 @@ export default function Profile(): JSX.Element {
   return (
     <Grid container>
       <Paper className={classes.profileContainer}>
-        <img className={classes.coverImage} src={coverPic} alt="Cover Photo" />
+        <img className={classes.coverImage} src={profile?.coverPic} alt="Cover Photo" />
         <Grid container className={classes.basicInfoContainer} direction="column" alignItems="center">
-          <img className={classes.profilePic} src={profilePic} alt="Profile Pic" />
+          <img className={classes.profilePic} src={profile?.profilePic} alt="Profile Pic" />
           <Typography variant="h4">
-            {firstName} {lastName}
+            {profile?.firstName} {profile?.lastName}
           </Typography>
-          <Typography variant="subtitle1">{introduction}</Typography>
-
+          <Typography variant="subtitle1">{profile?.introduction}</Typography>
           <Typography color="primary" variant="subtitle2">
-            <LocationOnIcon fontSize="small" className={classes.locationIcon} /> {address}
+            <LocationOnIcon fontSize="small" className={classes.locationIcon} /> Toronto,ON
```

Comment:
Can we use the location for the sitter's profile rather than hardcoding it here? :)

Reply:
Fixed